samples/sensor/adxl345/src/main.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+
+void main(void)
+{
+	struct sensor_value accel[3];
+
+	const struct device *dev = DEVICE_DT_GET_ANY(adi_adxl345);
+
+	if (dev == NULL) {
```

Comment:
dev will never be null,  replace this check with a `device_is_ready(dev)` check.  See other samples/sensors for example.